csharp/README.md

```diff
@@ -191,9 +171,9 @@ For example:
 python allTests.py --framework=net6.0
 ```
 
-And to run test build against .NET Core 3.1:
+And to run test build against .NET 7:
```

Comment:
It's not clear what the previous line:
```
python allTests.py --framework=net6.0
```

does.

Execute the tests with the .NET Framework 4.5 included in .NET 6.0? Sounds unlikely.